apps/backend/package.json

```diff
@@ -23,6 +23,7 @@
     "@auto-drive/s3": "workspace:*",
     "@auto-files/rpc-apis": "workspace:*",
     "@autonomys/asynchronous": "^1.5.15",
+    "@autonomys/auto-consensus": "^1.5.18",
```

Comment:
I would update the rest of the packages too as sometimes there will be resolution issues with multiple versions.

Reply:
I'd prefer update them in the feature branch once, for avoiding intermediary conflicts merging